gcc/config/riscv/t-zephyr

```diff
@@ -72,6 +76,10 @@ march=rv32e_zicsr_zifencei/mabi=ilp32e \
 march=rv32em_zicsr_zifencei/mabi=ilp32e \
 march=rv32emc_zicsr_zifencei/mabi=ilp32e \
 march=rv32emc_zicsr_zifencei_zba_zbb_zbc_zbs/mabi=ilp32e \
+march=rv32emc_zicsr/mabi=ilp32e \
+march=rv32emc_zicsr_zba_zbb_zbc_zbs/mabi=ilp32e \
+march=rv32emc_zicsr_zicntr_zba_zbb_zbc_zbs/mabi=ilp32e \
+march=rv32emc_zicsr_zicntr_zba_zbb_zbc_zbs_zcb/mabi=ilp32e \
```

Comment:
Here you are adding `rv32emc` multi-lib variants without `Zifencei` (i.e. support for `IFENCE.I`) -- is this intentional?

I am asking because `IFENCE.I` used to be part of the mandatory base instruction set, and generally speaking, it is hard to imagine a processor implementation without one.

Note that there already exist `rv32emc_zicsr_zifencei` variants if you are targeting processors with Zifencei support.

Reply:
Yes, adding variants without `Zifencei` is the main purpose of this PR.

I'm also adding variants with `zcb` and `zicntr` as those extensions were not used in Zephyr SDK yet, so calling this change "add rv32emc configurations without Zifencei" would not convey full scope here, that's why I picked such generic commit title.